src/azul/indexer/document.py

```diff
@@ -210,6 +210,7 @@ def __str__(self) -> str:
 class FieldType(Generic[N, T], metaclass=ABCMeta):
     shadowed: bool = False
     es_sort_mode: str = 'min'
+    unmapped_type: Optional[str] = None
```

Comment:
```suggestion
    es_type: Optional[str] = None
```

We will be able to use this to pre-populate the mappings.